pkgs/top-level/aliases.nix

```diff
@@ -2230,6 +2230,8 @@ mapAliases {
   z3_4_8 = throw "'z3_4_8' has been removed in favour of the latest version. Use 'z3'."; # Added 2025-05-18
   zabbix50 = throw "'zabbix50' has been removed, it would have reached its End of Life a few days after the release of NixOS 25.05. Consider upgrading to 'zabbix60' or 'zabbix70'.";
   zabbix64 = throw "'zabbix64' has been removed because it reached its End of Life. Consider upgrading to 'zabbix70'.";
+  zeal-qt5 = throw "zeal-qt5 has been replaced by zeal"; # Added 2025-08-31
+  zeal-qt6 = throw "zeal-qt6 has been replaced by zeal"; # Added 2025-08-31
```

Comment:
It would be better to use a warn alias first

Reply:
Example:

```
brasero-original = lib.warnOnInstantiate "Use 'brasero-unwrapped' instead of 'brasero-original'" brasero-unwrapped;
```

Reply:
I added the warning for `zeal-qt6` . For the qt5 version, is it preferred to keep it around for a while before removing it or should I keep the `throw` on that one?

Reply:
I would prefer both being warns and backporting to 25.05 stable

Reply:
I can do that change, but just to double check that I understood correctly:

- Keep the `zeal-qt5` version in `all-packages.nix`, but point it to the `pkgs/by-name`
- and do the same type of warning as for the zeal-qt6

Reply:
Was thinking more along the lines of auto-migrating:

```
zeal-qt5 = lib.warnOnInstantiate "'zeal-qt5' has been removed from nixpkgs. Please use 'zeal' instead" zeal; # Added 2025-08-31
```